hedera-node/hedera-token-service/src/test/java/com/hedera/node/app/service/token/AliasUtilsTest.java

```diff
@@ -170,19 +170,19 @@ void isSerializedProtoKeyReturnsFalseForEvmAddress() {
     @Test
     void extractShardFromAddressAliasReturnsCorrectValue() {
         var alias = Bytes.wrap(HexFormat.of().parseHex(TEST_NON_ZERO_SHARD_ALIAS));
-        assertEquals(1, AliasUtils.extractShardFromAddressAlias(alias));
+        assertEquals(1, AliasUtils.extractShardFromAddressAlias(alias, 10));
     }
 
     @Test
     void extractShardFromAddressAliasReturnsZeroForZeroShard() {
         var alias = Bytes.wrap(HexFormat.of().parseHex(TEST_ENTITY_NUM_ALIAS));
-        assertEquals(0, AliasUtils.extractShardFromAddressAlias(alias));
+        assertEquals(10, AliasUtils.extractShardFromAddressAlias(alias, 10));
     }
 
     @Test
     void extractRealmFromAddressAliasReturnsZeroForZeroRealm() {
         var alias = Bytes.wrap(HexFormat.of().parseHex(TEST_ENTITY_NUM_ALIAS));
-        assertEquals(0L, AliasUtils.extractRealmFromAddressAlias(alias));
+        assertEquals(20L, AliasUtils.extractRealmFromAddressAlias(alias, 20));
     }
```

Comment:
Should `extractRealmFromAddressAlias(...)` even exist if we're removing the shard/realm encoding from the alias? At least the test name should probably change

Reply:
You are right.  Once it was decided that the shard and realm for long zero addresses should always be 0, there is no point to these methods and unit tests.  Removed